PowerShell/BuildingBlocks/Authentication/GetTicketForCustomerOrWorkgroup.ps1

```diff
@@ -0,0 +1,17 @@
+﻿# This script shows how to get an authentication ticket
+# 1) Scoped to a customer
+# 2) Scoped to a workgroup
+
+# Authenticate
+$creds = Get-Credential
+$ticket = Get-BT_Ticket -Credentials $credentials -ServiceType BitTitan
```

Comment:
Hmm, I think you mean $creds instead of $credentials when you're retrieving the ticket :-)

Reply:
Gah - I might have changed it after testing it...